translating nn and nb

ghveem opened the pull request:
several new nn and nb

ghveem commented:
Should   "credits": include name of translator like 
  "credits": "Thomas Bedin", ?

pjanik commented:
Thank you. 
Translated models need i18nMetadata file, so students can see a menu with available languages.
I've added this file to previously translated model in this commit: https://github.com/concord-consortium/lab-interactives-site/commit/10a0b8479b38b7e28a780c7cfc81ee7a191c62f7
Could you follow this pattern and add the necessary metadata files? 
There's a readme section describing it: https://github.com/concord-consortium/lab-interactives-site/blob/master/developer-doc/localization.md
If something is unclear, let me know. 
Do you have a local copy of lab-interactives-site working?

> Should "credits": include name of translator like "credits": "Thomas Bedin",

I don't think so. "credits" specifies the content of the credits dialog. You can see it when you click on Concord Consortium logo in the lower-left corner.

ghveem commented:
Updated i18nmetadata in fresh files (translations) and updated the metadata-files with information about the translated files. 
I have not tried a local running copy.

pjanik commented:
Thanks. I think the English versions are missing "i18nMetadata" value. It's necessary to be able to see the flag button and be able to switch to another language.
Did you try to run a local copy? Basic instructions can work and there's also solution based on docker (one more tool, but more bulletproof). With this amount of changes, it'd be useful to see if things work correctly and iterate here faster (we're not actively working on this project at the moment).

ghveem commented:
I'm sorry. This is not ready or ment to be a PR. I just tried to sync our version from your master.
